k8s/kops/testground-daemon/deployment.yml

```diff
@@ -0,0 +1,82 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: testground-daemon
+  labels:
+    app: testground-daemon
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: testground-daemon
+  template:
+    metadata:
+      labels:
+        app: testground-daemon
+    spec:
+      serviceAccountName: testground-daemon
+      hostPID: true
+      hostNetwork: true
+      dnsPolicy: ClusterFirstWithHostNet
+      nodeSelector:
+        testground.node.role.infra: "true"
+      containers:
+      - name: goproxy
+        image: iptestground/goproxy:2.0.2
+        ports:
+        - containerPort: 8081
+          hostPort: 8081
+        volumeMounts:
+          - name: envtoml
+            mountPath: /root/testground/.env.toml
+            subPath: .env.toml
+          - name: efs-pvc
+            mountPath: "/go"
+        resources:
+          requests:
+            memory: 700Mi
+            cpu: 500m
+          limits:
+            memory: 1000Mi
+            cpu: 700m
+      - name: testground-daemon
+        image: 909427826938.dkr.ecr.eu-west-2.amazonaws.com/bloxico-testground:eks_26_july
```

Comment:
How did it end up in that ECR? Is it publicly available? Do we have a script for populating ECR?

Reply:
It is publicly available, it has to be publicly available. We do not have a script for populating yet, this was only moved there by me so you could run initial tests and verify whether it works.

Reply:
Where was it moved from? What version of testground does it refer to? Is it https://github.com/testground/testground/pull/1350?

Reply:
It was moved to pull from ECR, rather than the public testground dockerhub repo, where the images have initially been pulled from.
And I will have to get back to you on that, need to check with the team.

Reply:
The images were built from the code in the https://github.com/testground/testground/pull/1350.

Reply:
Which commit? And just to clarify, they were built on one of your machines, right?

Reply:
eks_v1.0 was/is `5bd117c648bc12562c3922ee3d6320ce92b3e57c` 
A new version will be built now with the hash `80c5aca36114de067c33c8718cca95ef16db4c06`

And yes, they were built on one of our machines.

Reply:
Can you add image build/publish instructions to the README? And a link to the specific testground commit as a comment next to where we specify the image here?

Reply:
Added a guide to the README `Advanced steps` section.